pymodbus/repl/client/mclient.py

```diff
@@ -109,107 +109,114 @@ def read_coils(self, address, count=1, **kwargs):
             return {"function_code": resp.function_code, "bits": resp.bits}
         return ExtendedRequestSupport._process_exception(resp)
 
-    def read_discrete_inputs(self, address, count=1, **kwargs):
+    def read_discrete_inputs(self, address, count=1, unit=0, **kwargs):
         """Read `count` number of discrete inputs starting at offset `address`.
 
         :param address: The starting address to read from
         :param count: The number of coils to read
+        :param unit: Modbus slave unit ID
         :param kwargs:
         :return: List of bits
         """
         resp = super().read_discrete_inputs(  # pylint: disable=no-member
-            address, count, **kwargs
+            address, count, unit, **kwargs
         )
         if not resp.isError():
             return {"function_code": resp.function_code, "bits": resp.bits}
         return ExtendedRequestSupport._process_exception(resp)
 
     @handle_brodcast
-    def write_coil(self, address, value, **kwargs):
+    def write_coil(self, address, value, unit=0, **kwargs):
         """Write `value` to coil at `address`.
 
         :param address: coil offset to write to
         :param value: bit value to write
+        :param unit: Modbus slave unit ID
```

Comment:
Please use identical description for unit in all places, using different descriptions are confusion.

I prefer this description overall.

Reply:
Can you be please more specific? Apart from unit iD vs. unit ID, I did not see the difference. Thanks

Reply:
???? You did that in an earlier commit.